js/components/interface/query/queryBuilder.js

```diff
@@ -767,6 +797,17 @@ define(function (require) {
                 headersColumns: headersDatasourceFormat
               });
 
+              if (that.props.sorterColumns !== undefined) {
+                that.sorterColumn = that.getSorterColumn(that.props.sorterColumns, columnsToShow);
```

Comment:
@ddelpiano I saw another change that made this plural (columns) but here it's still singular? I don't mind either way I am just wondering if this breaks everything, and also why one is plural if there's more than one of those fields. Also appears below so this comment applies to all references.

Reply:
that was intended to be like that, I am extracting from the array of sorterColumns the single one that should drive the order. I triple checked that when I did it initially as well, if you don't mind the names that can go as it is, differently I can quickly change it.

Reply:
it's a bit confusing, change it if you can think of a better name, nothing comes to my mind I think you are in a better position to come up with something clearer